src/libprojectM/libprojectM.xcodeproj/project.pbxproj

```diff
@@ -921,6 +921,7 @@
 				GCC_WARN_UNUSED_FUNCTION = YES;
 				GCC_WARN_UNUSED_VARIABLE = YES;
 				HEADER_SEARCH_PATHS = (
+					"$(SRCROOT)/../projectM-sdl/projectM-SDL",
```

Comment:
I don't think it's right that libprojectM should know anything about projectM-SDL